sonar/modules/deposits/jsonschemas/deposits/deposit-v1.0.0_src.json

```diff
@@ -560,25 +558,38 @@
               "type": "string",
               "minLength": 1
             },
-            "year": {
-              "title": "Year",
-              "type": "string",
-              "minLength": 1
-            },
             "volume": {
               "title": "Volume",
               "type": "string",
               "minLength": 1,
+              "pattern": "^[0-9]+$",
```

Comment:
from my experience, we coud have many unsuspected values in these kin of fields : `January`, `extra vol.1`, ....
If you force a pattern here, users will try to add this info into another inappropriate field.

Reply:
As discussed with @pronguen, at this time we will keep this constraint